cpu/xmc1000/include/periph/usic.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2015 Sebastian Sontberg <sebastian@sontberg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup         cpu_xmc1000
+ * @{
+ *
+ * @file
+ * @brief           Code shared by peripheral drivers using the USIC
+ *
+ * @author          Sebastian Sontberg <sebastian@sontberg.de>
+ *
+ * @}
+ */
+
+#ifndef USIC_H_
+#define USIC_H_
+
+#include "cpu.h"
+#include "periph/gating.h"
+#include "periph/usic_defs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup cpu_xmc1000_usic USIC configuration
+ * @ingroup cpu_xmc1000
+ * @{
+ */
+
+/**
+ * @brief   USIC channel initialization function
+ * @{
+ */
+static inline void usic_init(const usic_channel_t *usic_ch,
```

Comment:
I think it would make sense to de-inline this function for improved code-size. I would suggest to create a `cpu/xmc1000/periph/usic.c` and put the prototype to `cpu/xmc1000/include/periph_cpu.h`